tests/system/action/motion_statute_paragraph/test_sort.py

```diff
@@ -67,3 +68,36 @@ def test_sort_another_section_db(self) -> None:
         )
         self.assert_status_code(response, 400)
         assert "Additional db_instances found." in response.json["message"]
+
+    def test_sort_no_permissions(self) -> None:
+        self.base_permission_test(
+            {
+                "motion_statute_paragraph/31": {
+                    "meeting_id": 1,
+                    "title": "title_loisueb",
+                },
+                "motion_statute_paragraph/32": {
+                    "meeting_id": 1,
+                    "title": "title_blanumop",
+                },
+            },
```

Comment:
See https://github.com/OpenSlides/openslides-backend/pull/603#discussion_r602177045

Reply:
Add permission_test_model in setup.